bpf/process/bpf_process_event.h

```diff
@@ -5,7 +5,6 @@
 #define _BPF_PROCESS_EVENT__
 
 #define ENAMETOOLONG  36 /* File name too long */
-#define PATH_MAP_SIZE 4096
 
```

Comment:
Did you run `make format`?

I'm getting the following diff:
```
git diff
diff --git a/bpf/process/bpf_process_event.h b/bpf/process/bpf_process_event.h
index ab18561c..e41b95ab 100644
--- a/bpf/process/bpf_process_event.h
+++ b/bpf/process/bpf_process_event.h
@@ -4,7 +4,7 @@
 #ifndef _BPF_PROCESS_EVENT__
 #define _BPF_PROCESS_EVENT__
 
-#define ENAMETOOLONG  36 /* File name too long */
+#define ENAMETOOLONG 36 /* File name too long */
 
 struct bpf_map_def __attribute__((section("maps"), used)) buffer_heap_map = {
        .type = BPF_MAP_TYPE_PERCPU_ARRAY,
```

It seems to me that this is the reason for https://github.com/cilium/tetragon/runs/7948365267?check_suite_focus=true#step:9:14, which seems unrelated to https://github.com/cilium/tetragon/issues/344

Reply:
@kkourt I did but just tried to fix the code that I introduced in this PR and on #225 to reduce noise and ignore the rest! cause every file you touch clang-format may complain then checkpatch will complain again... hard to get it right.

Reply:
@tixxdz yap, it's hard to get right.

I've tried to fix the issues, and it seems to work:  https://github.com/cilium/tetragon/pull/345.

Here's the diff:

diff
```
diff --git a/bpf/lib/bpf_events.h b/bpf/lib/bpf_events.h
index d5a4eac5..6e256247 100644
--- a/bpf/lib/bpf_events.h
+++ b/bpf/lib/bpf_events.h
@@ -86,9 +86,8 @@ static inline __attribute__((always_inline)) __u32 get_task_pid_vnr(void)
        thread_pid_exists = bpf_core_field_exists(task->thread_pid);
        if (thread_pid_exists) {
                probe_read(&pid, sizeof(pid), _(&task->thread_pid));
-               if (!pid) {
+               if (!pid)
                        return 0;
-               }
        } else {
                struct pid_link link;
                int link_sz = bpf_core_field_size(task->pids);
diff --git a/bpf/process/bpf_process_event.h b/bpf/process/bpf_process_event.h
index ab18561c..e41b95ab 100644
--- a/bpf/process/bpf_process_event.h
+++ b/bpf/process/bpf_process_event.h
@@ -4,7 +4,7 @@
 #ifndef _BPF_PROCESS_EVENT__
 #define _BPF_PROCESS_EVENT__
 
-#define ENAMETOOLONG  36 /* File name too long */
+#define ENAMETOOLONG 36 /* File name too long */
 
 struct bpf_map_def __attribute__((section("maps"), used)) buffer_heap_map = {
        .type = BPF_MAP_TYPE_PERCPU_ARRAY,
```
Could you please pick up the patches from #345 and push them in this PR?

Reply:
Alright @kkourt sure will do much appreciated!